app.rb

```diff
@@ -0,0 +1,140 @@
+require_relative './person'
+require_relative './book'
+require_relative './rental'
+require_relative './student'
+require_relative './teacher'
+require_relative './classroom'
+require_relative './nameable'
+
+class App
+  def initialize
+    @books = []
+    @people = []
+    @rentals = []
+  end
+
+  def create_book
+    puts 'Enter book\'s title'
+    title = gets.chomp
+    puts 'Enter author\'s name'
+    author = gets.chomp
+    book = Book.new(title, author)
+    @books << book
+    puts "Created #{book.title} by #{book.author}"
+  end
+
+  def list_books
+    if @books.empty?
+      puts 'There are no books available'
+    else
+      @books.each do |book|
+        puts "Title: '#{book.title}', Author: '#{book.author}'"
+      end
+    end
+  end
+
+  def list_people
+    if @people.empty?
+      puts 'There\'s no one here ATM!'
+    else
+      @people.each do |person|
+        puts "Name: #{person.name} Age: #{person.age} ID: #{person.id}"
```

Comment:
- [OPTIONAL] Good job so far @AbdifatahYasin1, I noticed something that needs to be fixed. When I select option 2 `List all people` in the result there is something that you should fix:
  - for the student instead of listing the name of the students it shows the `grade` and not the name also for the age it shows the name and not the age 🙃 
  - for the teacher instead of listing the name of the teacher it shows the `specialization` and for the age it shows the name and not the age 🙃 

<img width="489" alt="Screenshot 2023-04-01 at 15 28 31" src="https://user-images.githubusercontent.com/74501523/229289433-8a765f48-75f1-4736-a7f1-ce73bef98bd0.png">